whylog/config/parsers.py

```diff
@@ -1,32 +1,42 @@
-import re
 from abc import ABCMeta, abstractmethod
 
 import six
+from frozendict import frozendict
+
+IMPORTED_RE = False
+
+try:
+    import regex
+except ImportError:
```

Comment:
It is not just because we want to be able to work without regex being installed, but the reader of this code cannot know why you did it like that. Please add a comment explaining your intention.